rel/config/config.exs

```diff
@@ -111,6 +111,11 @@ if Utils.Environment.exists?("BASIC_AUTH_USERNAME") do
     ]
 end
 
+# Configure CORS
+config :tv5, :corsica,
+  origins: System.get_env("CORS_ALLOWED_ORIGINS"),
```

Comment:
What is `CORS_ALLOWED_ORIGINS` exactly? A string of comma-separated origins? Eg. `"foo.com,bar.com"`?